### ISSUE TITLE
 How about the evaluation on COCO2017?

### ISSUE THREAD

**Issue by hellojialee** [kevinlin311tw/keras-openpose-reproduce#4]
Hi~ Dose the performance drops obvious on the coco 2017 dataset? I have trained my own model but the AP is about 0.5. Though, I change the learning rate and the numbers of iterations. 
Hoping for your advice. Thank you!

**Comment by kevinlin311tw** [comment]
I did not try that. Since COCO17 has more training data, maybe you could apply a more aggressive data augmentation, and more training iterations.
